### PR TITLE
Add instructions for configuring git on CUBIC

### LINKED ISSUE #158: How to set up GitHub credentials on a project user
Here's what I did yesterday for the PAFIN project user, where I'll be the main person working on the code.

1. `git config --global user.name <username>`
2. `git config --global user.email <email>`
3. Create fine-grained PAT on GitHub.
    - Resource owner is PennLINC
    - Longest expiration time I could select (366 days)
    - Read access to metadata
    - Read and Write access to code, commit statuses, and pull requests
8. `git config --global credential.helper store`
    - This will store the PAT in `~/.git-credentials`. It's plain text, but not an issue since we only have one repo.
10. `git clone https://github.com/PennLINC/affective-instability.git`
    - Must be HTTPS
12. Put in the username
13. Put in the PAT
14. Now you can make commits without needing to enter your credentials

### PR DESCRIPTION
**Pull request by tsalo**
Closes #158.

Changes proposed:

- Distinguish project creation and access requests.
- Add instructions for configuring git on CUBIC.

### CODE REVIEW

File: docs/cubic/cubic.md [hunk 1]
```diff
@@ -36,12 +36,17 @@ You use your UPHS password to login. If you don't have access to cubic-sattertt,
...
+First you need to fill out the data management document available [here](https://cbica-wiki.uphs.upenn.edu/wiki/images/Project_data_use_template.doc).
+If this link doesn't work for you, you can find this document on the CBICA Wiki: `Main Page > Research projects > 3 Access/New Project Creation > Project Creation Request`.
+This document will ask you for a number of details about your project, including the data's source and estimates about how much disk space you will need over a 6 month, 12 month, and 24 month period, and the estimated lifespan of the data ( 🤷).
```
Comment (singlesp) [inline docs/cubic/cubic.md:48]:
```suggestion
This document will ask you for a number of details about your project, including the data's source and estimates about how much disk space you will need over a 6 month, 12 month, and 24 month period, and the estimated lifespan of the data ( 🤷).
PennLINC is only charged for the amount of disk space _actually used_ on a project, not the amount you request at this stage. It is reasonable to be liberal in your disk space estimate at this stage, as it is a pain to upgrade later.
```

Comment (singlesp) [reply docs/cubic/cubic.md:48]: this wasn't made clear to me at first and I spent too much time fretting over getting disk space right

**Review by singlesp — CHANGES_REQUESTED** [review]
If you approve my suggestion - commit & merge.

File: docs/cubic/cubic.md (hunk 1 above)
Comment (tsalo) [reply docs/cubic/cubic.md:48]: I didn't know that. Definitely good to have in the docs.

**Pull request merged**